playbooks/openstack/openshift-cluster/post-install.yml

```diff
@@ -0,0 +1,57 @@
+---
+- hosts: OSEv3
```

Comment:
So what should we use to access all cluster nodes? I've seen `nodes:masters:etcd` and `oo_all_hosts`. We can't use `all` because that would include `localhost` as well as the potential `lb` nodes.

Where are the `oo_*` groups set? Should I update the dynamic inventory to produce them?

Reply:
oo_all_hosts would be the correct one.

The groups are created by running an 'entry point' playbook.

These basically include playbooks/byo/openshift-cluster/initialize_groups.yml and playbooks/common/openshift-cluster/std_include.yml